classes/components/forms/dashboard/SubmissionFilters.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * @file classes/components/form/dashboard/SubmissionFilters.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2000-2023 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionFilters
+ *
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form to add and remove filters in the submissions dashboard
+ */
+
+namespace PKP\components\forms\dashboard;
+
+use APP\components\forms\FieldSelectIssues;
+use APP\core\Application;
+use APP\facades\Repo;
+use Illuminate\Support\LazyCollection;
+use PKP\category\Category;
+use PKP\components\forms\FieldAutosuggestPreset;
+use PKP\components\forms\FieldOptions;
+use PKP\components\forms\FieldSelectUsers;
+use PKP\components\forms\FormComponent;
+use PKP\context\Context;
+use PKP\security\Role;
+
+class SubmissionFilters extends FormComponent
+{
+    /**
+     * The maximum number of options in a field
+     * before it should be shown as an autosuggest
+     */
+    public const OPTIONS_MAX = 7;
+
+    public $id = 'submissionFilters';
+    public $action = FormComponent::ACTION_EMIT;
+
+    public function __construct(
+        public Context $context,
+        public array $userRoles,
+        public LazyCollection $sections,
+        public LazyCollection $categories
+    )
+    {
+        $this
+            ->addSectionFields()
+            ->addAssignedTo()
+            ->addIssues()
+            ->addCategories()
+        ;
+    }
+
+    protected function isManager(): bool
```

Comment:
`isManager` is misleading; `isManagerOrAdmin` would be more accurate. Maybe name the function for its purpose (`canAssign`) rather than how it works?